src/Android/Xamarin.Android/Samples/Analysis/LineOfSightGeoElement/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Name": "Line of Sight (GeoElement)",
+  "SampleName": "LineOfSightGeoElement",
+  "Description": "",
+  "Instructions": "", 
+  "Type": 0,
+  "RequiresOnlineConnection": true,
+  "RequiresOfflineData": false,
```

Comment:
Does this not require offline data? also will it not need "DataItemIds" and "DataFileNames"?